download_render_tools_from_golem_repository.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+
+
+display_usage(){
```

Comment:
To be consistent with our conventions in deployment please use the `function` keyword when defining a function and leave a space before the `{`.

Reply:
Done